### PR TITLE
librdb: add perf counters descriptions

### PR DESCRIPTION
**Pull request by Ved-vampir**
Sign-off-by: Alyona Kiseleva <akiselyova@mirantis.com>

Forgot this one

### CODE REVIEW

File: src/librbd/ImageCtx.cc [hunk 1]
```diff
@@ -237,34 +237,34 @@ namespace librbd {
...
-    plb.add_u64_counter(l_librbd_aio_flush, "aio_flush");
-    plb.add_time_avg(l_librbd_aio_flush_latency, "aio_flush_latency");
-    plb.add_u64_counter(l_librbd_snap_create, "snap_create");
-    plb.add_u64_counter(l_librbd_snap_remove, "snap_remove");
-    plb.add_u64_counter(l_librbd_snap_rollback, "snap_rollback");
-    plb.add_u64_counter(l_librbd_notify, "notify");
-    plb.add_u64_counter(l_librbd_resize, "resize");
-    plb.add_u64_counter(l_librbd_readahead, "readahead");
-    plb.add_u64_counter(l_librbd_readahead_bytes, "readahead_bytes");
+    plb.add_u64_counter(l_librbd_rd, "rd", "Iterate reads");
```
Comment (jdurgin) [inline src/librbd/ImageCtx.cc]: these three librbd_rd counters are for read() as well as read_iterate(). I'd just leave iterate out of the description

File: src/librbd/ImageCtx.cc [hunk 2]
```diff
@@ -237,34 +237,34 @@ namespace librbd {
...
+    plb.add_u64_counter(l_librbd_rd_bytes, "rd_bytes", "Data size in iterate reads");
+    plb.add_time_avg(l_librbd_rd_latency, "rd_latency", "Latency of iterate reads");
+    plb.add_u64_counter(l_librbd_wr, "wr", "Writes");
+    plb.add_u64_counter(l_librbd_wr_bytes, "wr_bytes", "Written data");
+    plb.add_time_avg(l_librbd_wr_latency, "wr_latency", "Write latency");
+    plb.add_u64_counter(l_librbd_discard, "discard", "Discardes");
```
Comment (jdurgin) [inline src/librbd/ImageCtx.cc]: typo: extra e

File: src/librbd/ImageCtx.cc [hunk 3]
```diff
@@ -237,34 +237,34 @@ namespace librbd {
...
+    plb.add_u64_counter(l_librbd_aio_wr, "aio_wr", "Async writes");
+    plb.add_u64_counter(l_librbd_aio_wr_bytes, "aio_wr_bytes", "Data size in async writes");
+    plb.add_time_avg(l_librbd_aio_wr_latency, "aio_wr_latency", "Latency of async writes");
+    plb.add_u64_counter(l_librbd_aio_discard, "aio_discard", "Async discards");
+    plb.add_u64_counter(l_librbd_aio_discard_bytes, "aio_discard_bytes", "Data size in async discards");
+    plb.add_time_avg(l_librbd_aio_discard_latency, "aio_discard_latency", "Latency of async discards");
+    plb.add_u64_counter(l_librbd_aio_flush, "aio_flush", "Async flushes");
+    plb.add_time_avg(l_librbd_aio_flush_latency, "aio_flush_latency", "Latency of async flushes");
+    plb.add_u64_counter(l_librbd_snap_create, "snap_create", "Snap creations");
+    plb.add_u64_counter(l_librbd_snap_remove, "snap_remove", "Snap removings");
```
Comment (jdurgin) [inline src/librbd/ImageCtx.cc]: s/removings/removals/

**Comment by Ved-vampir** [comment]
Changed

**Pull request merged**